IntegrationTemplates/AnalyticsAndSIEM/AnalyticsAndSIEM.py

```diff
@@ -0,0 +1,443 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+''' IMPORTS '''
+from typing import cast, Any, Dict, Tuple, List, AnyStr, Optional
+from xml.etree import ElementTree
+
+import requests
+import urllib3
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+''' GLOBALS/PARAMS '''
+INTEGRATION_NAME: str = 'Analytics & SIEM Integration'
+# lowercase with `-` dividers
+INTEGRATION_NAME_COMMAND: str = 'analytics-and-siem'
+# No dividers
+INTEGRATION_NAME_CONTEXT: str = 'AnalyticsAndSIEM'
+
+
+class Client:
```

Comment:
What do you think about making all the suffixs as class attributes?
It will help making sure that they are in the correct format